content/en/docs/installing-on-linux/introduction/ha-configuration.md

```diff
@@ -45,26 +45,38 @@ You must create a load balancer in your environment to listen (also known as lis
 
 {{< tab "For users with good network connections to GitHub" >}}
 
-Download KubeKey from [GitHub Release Page](https://github.com/kubesphere/kubekey/releases/tag/v1.0.0) or use the following command directly.
+Download KubeKey from its [GitHub Release Page](https://github.com/kubesphere/kubekey/releases) or use the following command directly.
 
 ```bash
-wget https://github.com/kubesphere/kubekey/releases/download/v1.0.0/kubekey-v1.0.0-linux-amd64.tar.gz -O - | tar -xz
+curl -sfL https://get-kk.kubesphere.io | VERSION=v1.0.1 sh -
```

Comment:
Is this download link counted by github? @Sherlock113 @pixiake

Reply:
Yes，the `downloadKubekey.sh` in release can be counted。
https://somsubhra.com/github-release-stats/?username=kubesphere&repository=kubekey

Reply:
Yes.

From v1.0.1, the K8s binary will be downloaded from Google drive if users do not set `KKZONE=cn` (for users who are able to access GitHub and Google). Otherwise, the K8s binary will be downloaded from QingStor (for users who are not able to access GitHub and Google).